Taarafo.Core/Services/Foundations/PostImpressions/IPostImpressionService.cs

```diff
@@ -13,6 +14,6 @@ public interface IPostImpressionService
     {
         ValueTask<PostImpression> AddPostImpressions(PostImpression postImpression);
         IQueryable<PostImpression> RetrieveAllPostImpressions();
-
+        ValueTask<PostImpression> RemovePostImpressionByIdAsync(Guid postId, Guid profileId);
```

Comment:
Change to ValueTask<PostImpression> RemovePostImpressionAsync(PostImpression postImpression);